docs/topics/wasm/wasm-overview.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (title: Kotlin Wasm)
+
+> Kotlin Wasm is [Experimental](components-stability.md).
+> It may be changed at any time. Use it only for evaluation purposes.
+> 
+> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+[WebAssembly (Wasm)](https://webassembly.org) is a binary format for a virtual machine that can run in web browsers.
+It is platform-independent because it runs on its own virtual machine.
+Wasm is designed to be fast and secure, and it can compile code from various programming languages, including Kotlin.
+
+Kotlin/Wasm is a new compilation target for Kotlin. 
+With Kotlin/Wasm, you can take advantage of the benefits of WebAssembly to write web applications that are capable of running on a variety of environments and devices.
+
+Kotlin features a new experimental Kotlin/Wasm compiler that provides several advantages:
+
+* Faster compilation speeds than the deprecated `wasm32` Kotlin/Native target since Kotlin/Wasm doesn't rely on LLVM.
+* Improved interoperability with JavaScript and better integration with browsers, thanks to the [Wasm garbage collection](https://github.com/WebAssembly/gc) feature.
+* Potentially faster application startup compared to Kotlin/JS and JavaScript because Wasm has a compact and easy-to-parse bytecode.
+* Improved application runtime performance compared to Kotlin/JS and JavaScript because Wasm is a statically typed language.
+
+You can use Kotlin standard library (`stdlib`) and test library (`kotlin.test`) for Kotlin/Wasm out of the box.
+
+[Learn more about Kotlin/Wasm in our YouTube video](https://www.youtube.com/watch?v=-pqz9sKXatw).
```

Comment:
This video is a bit old. Let's omit it now and add a new one soon we get a new one.

Reply:
IDK, do you think it's suitable here? cc @dromanov-jb